docker-compose.yaml

```diff
@@ -0,0 +1,94 @@
+version: "3.7"
```

Comment:
with `docker compose` (the one that we should use) this line is deprecated. Just remove it